### PR TITLE
fix: rad streams in echam and hyper off on DKRZ

### PR DESCRIPTION
**Pull request by pgierz**
Rad streams where not properly moved to next compute job, led to model crash after first year.

Also, hyper threading is off now on MISTRAL by default, as it induces longer wall times.

@mandresm, can you please put this in develop too?

### CODE REVIEW

File: configs/machines/mistral.yaml [hunk 1]
```diff
@@ -6,14 +6,14 @@ additional_flags: "--mem=0"
...
-                hyperthreading_flag: "--ntasks-per-core=1"
-        "*":
+        "1":
                 hyperthreading_flag: ""
+        1:
+                hyperthreading_flag: ""
+        True:
+                hyperthreading_flag: ""
+        "*":
+                hyperthreading_flag: "--ntasks-per-core=1"
```
Comment (mandresm) [inline configs/machines/mistral.yaml]: Have you tried this lines on `mistral`? My guess is that they will break as `1` and `True` are considered as the same once evaluated by the parser and tagged as duplicates. `"1"` and `True` should cause no issue.

Comment (pgierz) [reply configs/machines/mistral.yaml]: Just tried it now again, it did not seem to complain....but it also didn't turn the flag on, I'll send you another commit in a bit.

Comment (mandresm) [reply configs/machines/mistral.yaml]: The flag does not need to be on for it to complain. So I would leave it as it is (including the `1`).

Comment (pgierz) [reply configs/machines/mistral.yaml]: OK, I fixed it now...

Also, the "tidy done" that I had was wrong earlier, the "maybe resubmit" kills off any steps that might happen afterward.

**Pull request merged**